Dockerfile

```diff
@@ -1,29 +1,33 @@
 FROM node:18-alpine as development
 
+USER node
+
 WORKDIR /app
 
-COPY package*.json ./
-COPY tsconfig*.json ./
-COPY src ./src
-COPY /prisma ./prisma/
+COPY --chown=node:node package*.json ./
+COPY --chown=node:node tsconfig*.json ./
+COPY --chown=node:node src ./src
+COPY --chown=node:node prisma ./prisma/
 
 RUN npm ci
 
 RUN npm run build
 
 FROM node:18-alpine as production
```

Comment:
here you have `as production` and above `as development` which is kind of misleading. I would name the above `as builder` and here `as app`